addons/mass_mailing/data/mailing_data_templates.xml

```diff
@@ -15,7 +15,8 @@
                     <b><t t-out="mailing.ab_testing_mailings_count - 1"/> other versions</b> from the same campaign.
                 </p>
                 <p>
-                    <t t-if="mailing.ab_testing_winner_selection == 'manual'">Don't forget to send your prefered version</t>
+		    <t t-if="mailing.ab_testing_winner_selection == 'manual'">Don't forget to send your prefered version</t>
+		    <t t-elif="not mailing.ab_testing_schedule_datetime">Since the date and time for this test has not been scheduled, don't forget to manually send your preferred version.</t>	
```

Comment:
Since we can't really put a random date for `ab_testing_schedule_datetime` during the upgrade, it should do the job I suppose. The message could also include the possibility to select also a datetime. Sending the version manually is only one of the possibilities here